src/components/TodoCount.js

```diff
@@ -0,0 +1,37 @@
+export default function TodoCount({ onFilter }) {
+  const $todoCountContainer = document.querySelector('.count-container');
+
+  const $filterButtons = {
+    all: $todoCountContainer.querySelector('.all'),
+    active: $todoCountContainer.querySelector('.active'),
+    completed: $todoCountContainer.querySelector('.completed'),
+  };
+
+  $todoCountContainer.addEventListener('click', (event) => this.changeFilterButtonStatus(event));
+
+  this.render = (count) => {
+    const todoCount = $todoCountContainer.querySelector('strong');
+    todoCount.innerText = count;
+  };
+
+  this.changeSelection = (filterName) => {
+    Object.keys($filterButtons).map((key) => $filterButtons[key].classList.remove('selected'));
+    $filterButtons[filterName].classList.add('selected');
+  };
+
+  this.initFilterButtonStatus = () => {
+    const filterStatus = window.location.hash.replace(/#/, '');
+    if (filterStatus === '') return;
+    this.changeSelection(filterStatus);
+  };
+
+  this.changeFilterButtonStatus = (event) => {
+    const { target } = event;
+
+    if (target.tagName !== 'A') return;
+
+    const filterName = target.classList[0];
```

Comment:
```suggestion
const [type] = target.classList;
```

이 방법을 사용해서 매직넘버 0을 없애보는 건 어떠실까요 ?

Reply:
생각지도 못했는데, 이런 식으로 매직 넘버를 없앨 수도 있겠네요. 감사합니다!